kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/order/OrderRepository.java

```diff
@@ -266,25 +257,14 @@ private boolean isCachedOpenOrderEquals(String orderId) {
 		return false;
 	}
 
-	private boolean isCurrentPendingOfferIdEquals(String offerId) {
-		if (offerRepository.getPendingOffer() != null) {
-			final Offer pendingOffer = offerRepository.getPendingOffer().getValue();
-			if (pendingOffer != null) {
-				return pendingOffer.getId().equals(offerId);
-			}
-		}
-		return false;
-	}
-
 	@Override
 	public void cancelOrder(@NonNull final String offerID, @NonNull final String orderID,
 		@Nullable final KinCallback<Void> callback) {
-		decrementCount();
+		removeCachedOpenOrderByID(orderID);
```

Comment:
why do you call removeCachedOpenOrderByID both here and in remoteData.cancelOrder onResponse()?

Reply:
a mistake, thanks!